File: sistema_facturacion_api/sistema_facturacion_api.Data/DTOs/TblFacturasDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace sistema_facturacion_api.Data.DTOs
{
    public class TblFacturasDTO
    {
        public int? Id { get; set; }
        public int? EmpresaId { get; set; }
        public int? UsuarioId { get; set; }
        public DateTime? FechaDeCreacion { get; set; }
        public string? LineaDePago { get; set; }
        public decimal? SubTotal { get; set; }
        public decimal? Descuento { get; set; }
        public decimal? IVA { get; set; }
        public decimal? TotalPagado { get; set; }
        public int? FormaDePagoId { get; set; }
    }
}

File: sistema-facturacion-front/src/app/models/dtos/tbl-facturas-dto.ts

export class TblFacturasDTO{

  id: number;
  empresaId: number;
  usuarioId: number;
  fechaDeCreacion: Date;
  lineaDePago: string;
  subTotal: number;
  descuento: number;
  iva: number;
  totalPagado: number;
  formaDePagoId: number;

  constructor(){
    this.id = 0;
    this.empresaId = 0;
    this.usuarioId = 0;
    this.fechaDeCreacion = new Date();
    this.lineaDePago = '';
    this.subTotal = 0;
    this.descuento = 0;
    this.iva = 0;
    this.totalPagado = 0;
    this.formaDePagoId = 0;
  }
}
